### ISSUE TITLE
Display battery level?

### ISSUE THREAD

**Issue by imalightbulb** [prasanthrangan/hyprdots#54]
First of all thanks a lot for the config. It's my first time on hyprland and I'm wondering how do I modify waybar so that it displays the battery level? (or is it somewhere and I didn't notice it?)

**Comment by Ittyy-Winter** [comment]
In my case I add it in waybar
![swappy-20230727_133905](https://github.com/prasanthrangan/hyprdots/assets/96482477/d3d31e18-bc6f-4466-a37f-537f5830bbc4)

You have to config 3 parts in /home/Your_USERNAME/.config/waybar/
1. config.ctl  //where you want to locate it
2. style.css   //put in  #battery
3. config.jsonc //what you want it to look like

here is my [config.jsonc](https://github.com/prasanthrangan/hyprdots/files/12180142/battery.txt)

**Comment by vrd1** [comment]
> In my case I add it in waybar ![swappy-20230727_133905](https://user-images.githubusercontent.com/96482477/256447620-d3d31e18-bc6f-4466-a37f-537f5830bbc4.png)
> 
> You have to config 3 parts in /home/Your_USERNAME/.config/waybar/
> 
>     1. config.ctl  //where you want to locate it
> 
>     2. style.css   //put in  #battery
> 
>     3. config.jsonc //what you want it to look like
> 
> 
> here is my [config.jsonc](https://github.com/prasanthrangan/hyprdots/files/12180142/battery.txt)

Can you share all 3 configs?
I tried everything you said but it doesn't show up.

**Comment by prasanthrangan** [comment]
Hi @vrd1 ,

For battery module `style.css` and `config.jsonc` is generated based on these modules -
https://github.com/prasanthrangan/hyprdots/blob/main/Configs/.config/waybar/modules/battery.jsonc
https://github.com/prasanthrangan/hyprdots/blob/main/Configs/.config/waybar/modules/style.css

So you just have to add `battery` the `config.ctl` to use it, for example -

```
0||top|( wlr/workspaces hyprland/window )|( clock )|( cpu memory battery ) ( network bluetooth pulseaudio pulseaudio#microphone custom/updates ) ( tray ) ( custom/wallchange custom/mode custom/wbar custom/cliphist custom/power )
```

**Comment by vrd1** [comment]
Hey,

I actually got it working fine the first time I installed your scripts. But after I updated my Arch the waybar wasn't working (i fixed that fairly quickly), and after that I am unable to get it to work again. 

Just to be clear I have:

1. battery added in my active bar in ```config.ctl```
2. copied battery module in ```config.jsonc```
3. and as I see in ```style.css``` #battery is in there by default

Let me know if I missed something.
